env0/resource_user_team_assignment.go

```diff
@@ -34,7 +33,7 @@ func (a *UserTeamAssignment) GetId() string {
 }
 
 func GetUserTeamAssignmentFromId(id string) (*UserTeamAssignment, error) {
-	splitUserTeam := strings.Split(id, "_")
+	splitUserTeam := lastSplit(id, "_")
```

Comment:
At the moment this is the only code where import is used with a combination of "user".
If more combinations are added need to be sure lastSplit is used.

Reply:
just to make sure - team id is some kind of guid, so it won't have `_` in it, so we won't have to consider this case as well?

Reply:
AFAIK yes - a uuid.
userid is not a uuid.